config/share_notify.yml.prod.sample

```diff
@@ -0,0 +1,3 @@
+---
+host: <%= ENV.fetch('APPS_DHH_PROD_SHARE_NOTIFY_HOST') {'https://staging.osf.io'}%>
```

Comment:
@mcritchlow   If share api is updated to use the similar way like active fedora and Huawei's way to load the yml, then I can just make it use different env variables like other yml.  @hweng  Could you bring it up to Adam or whoever work on share api?  Thanks